src/app.js

```diff
@@ -71,60 +71,88 @@ app.listen(port, () => {
 });
 
 app.post("/validate-organisation-type", (req, res) => {
-  const result = validateForm(req.body.organisationType, "/help-with-hint");
-  const renderOptions = {
-    showError: result.showError,
-  };
-  if (result.showError) {
-    res.render("organisationType.njk", renderOptions);
-  } else if (result.redirect) {
-    res.redirect(result.redirect);
+  const queryParams = req.query;
+  const editMode = queryParams.editMode;
+
+  if (editMode === "true") {
+    res.redirect("/confirmation-page");
```

Comment:
Ah i've tested this scenario and you're not able to save empty values on any of the form pages

Reply:
Functionally yes you're right, that is working but technically you're not doing any validation. That is working because it's radio buttons field and you can't put it empty.
If you compare your implementation here and the others one, you will see a difference about the validation.